packages/dialect-react/components/ApiContext/index.tsx

```diff
@@ -36,27 +36,25 @@ type PropsType = {
   dapp?: string; // base58 public key format
 };
 
+export type { WalletName };
```

Comment:
Also, does WalletName contain `Anchor` as well? I see that the casting went through, which is great.

Reply:
it's just a bit different in a new version, there's no such enum at all. I guess they did it in order to avoid compile time dep. So I created an alternative enum in our package